packages/lib/config/definitions.js

```diff
@@ -33,6 +33,13 @@ MasterGroup.define({
 	optional: true,
 	initial_value: 8443,
 });
+MasterGroup.define({
+	name: "address_bind",
+	title: "Address bind",
+	description: "The address to bind the master https/http server to.",
```

Comment:
Suggested rewording: `IP address to bind the HTTP and HTTPS ports on.`

Reply:
done